release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -175,6 +175,15 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-4-10-web-console-developer-perspective"]
+==== Developer perspective
+* You can now specify a name to a Service Binding connector in the *Topology* view while making a binding connection.
+* You can check if a pipeline is created and not started using the status decorator of the pod in the *Topology* view.
+* This enhancement gives you a better user experience when an application export is in progress a dialog box window or an alert will appear to view the logs, you can also cancel or restart the exporting process from this dialog box window.
+* You can now add your new Helm Chart Repository in the Developer Catalog by creating a new custom resource. Refer to the QuickStart guides in the Developer perspective to add a new *HelmChartRepo*.
```

Comment:
Is this about namespaced helm chart repos? Then I would suggest to call the exact resource at the end:

```suggestion
* You can now add your new Helm Chart Repository in the Developer Catalog by creating a new custom resource. Refer to the QuickStart guides in the Developer perspective to add a new *ProjectHelmChartRepository*.
```

Reply:
Yes @jerolimov it is about namespaced helm chart repos. I'll update the text with the exact resource.